arcgis-ios-sdk-samples/Display information/Sketch on the map/README.md

```diff
@@ -11,9 +11,7 @@ A field worker could annotate features of interest on a map (via the GUI) such a
 
 ## How to use the sample
 
-Choose which geometry type to sketch from one of the available buttons. Choose from points, multipoints, polylines, polygons, freehand polylines, and freehand polygons.
-
-Use the control panel to cancel the sketch, undo or redo changes made to the sketch and to save the sketch to the graphics overlay. There is also the option to select a saved graphic and edit its geometry using the Sketch Editor. The graphics overlay can be cleared using the clear all button.
+Tap the add button to choose a geometry for the Sketch Editor. Use the toolbar to undo or redo changes made to the sketch on the graphics overlay. The graphics overlay can be cleared using the clear all button.
```

Comment:
In terms of saving, it seems different platforms have different strategy.

- https://developers.arcgis.com/android/java/sample-code/sketch-editor/ , Unfortunately the Pixel simulator doesn't boot for me so I cannot verify, but the Android screenshot does not have a save button.
- https://developers.arcgis.com/qt/qml/sample-code/sketch-on-map/ , Qt has the latest sample and has a save button
- `designs/SketchOnMap` mentions saving and a save button is included in the pane

I don't think it's worth making the UI overcomplicated, but let's leave that to the weekly review.